src/quo2/components/network_amount.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.network-amount
+  (:require [clojure.string :as string]
+            [quo.react-native :as rn]
+            [quo2.components.icon :as icon]
+            [quo2.components.text :as text]
+            [quo2.foundations.colors :as colors]
+            [status-im.i18n.i18n :as i18n]))
+
+(def ^:private sizes
+  {:icon {:small 12 :medium 16 :large 20}
+   :container {:small 32 :medium 32 :large 48}
+   :label {:small :label :medium :paragraph-2 :large :paragraph-1}
+   :amount-text {:small :paragraph-2 :medium :paragraph-1 :large :heading-2}
```

Comment:
hi, why all these sizes? According to design it only has one size.

Reply:
Thanks. The issue #13556 states "don't forget to add sizing to the issue". I assumed this meant to also create sizes. 

The `:small` size is now per the design, the other sizes I made equivalent to the `quo2.components.group-avatar`. I now see that the [Group Avatar design](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=179%3A9760) indeed has different sizes, while the Network Amount has not. So "don't forget to add sizing to the issue" probably refers to something else and my assumption was mistaken.

Shall I remove `sizes` and hard code the size of the component as per the design?

Reply:
> don't forget to add sizing to the issue

Yes,I think it is different sizing. 
(@cammellos Please can you confirm if by sizing you meant complexity estimation?)

> Shall I remove sizes and hard code the size of the component as per the design?

Yes please

Reply:
Yes, sorry for the confusion, I meant estimation

Reply:
Done, only small size now. Screenshot:

![status-network-amount-ios](https://user-images.githubusercontent.com/5002921/186896306-310e1c87-f790-42c6-9ade-1a59a077525f.png)

I am not sure about the quality of the icon. The [quo2.components.icon/icon](https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/icon.cljs#L6) component makes it impossible to use a higher resolution image, since the provided size is appended to the icon name. [[issue-13557] Network breakdown component](https://github.com/status-im/status-mobile/pull/13781) uses bigger icons and shrinks them, using the old `icon` function. I thought about overloading the new `quo2.components.icon/icon` component so that a higher resolution image can be used, but not sure if that aligns with the reasoning behind the current implementation.

Also in [[issue-13557] Network breakdown component](https://github.com/status-im/status-mobile/pull/13781) the @2x and @3x icons are the same size. In this implementation they are following the convention that @3x is three times the size instead of two, but if the @3x icon is used the icon seems out of proportion for the network amount component and does not follow the design.

Can you let me know what the preferred approach is here?